presto-main/src/main/java/com/facebook/presto/sql/planner/plan/AggregationNode.java

```diff
@@ -298,9 +298,9 @@ public Aggregation(
                 @JsonProperty("signature") Signature signature,
                 @JsonProperty("mask") Optional<Symbol> mask)
         {
-            this.call = call;
-            this.signature = signature;
-            this.mask = mask;
```

Comment:
this looks like a separate commit

Reply:
You mean like extracting only this three line change to a separate commit? I think it logically belongs to `Simplify Optional handling in LocalExecutionPlanner` as we depend on this assertion in that commit.